lib/Executions.js

```diff
@@ -22,6 +21,41 @@ class Executions {
 
     return result
   };
+
+  /**
+   * Returns a list of executions summary, with filtering.
+   *
+   * @returns {Promise.<Execution[]>} The returned objects include the execution summary,
+   *                                  detailed objects with data about the execution.
+   */
+  async getExecutions () {
+    return this.requestManager.request({
+      actionUrl: this.executionsActionUrl + '?pageSize=1000&pageNum=1',
```

Comment:
maybe this function can support parameters that will control the `pageSize`, `pageNum` ?
we can leave that for a follow-up PR though, it's not a must for this.